frontend/src/components/CreateDataSourceExample.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <div class="w-full flex flex-col justify-start" :class="props.className">
+  <div class="flex flex-col justify-start w-full" :class="props.className">
```

Comment:
revert the changes that are unrelated to DM. We can always use another PR if there are things to be improved.